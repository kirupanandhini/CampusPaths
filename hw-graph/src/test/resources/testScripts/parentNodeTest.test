# Tests a graph with one parent node that has multiple children

# Create a graph
CreateGraph graph1

# Add several nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 n4
AddNode graph1 n5
AddNode graph1 n6

# Add several edges connecting n1 to other nodes
AddEdge graph1 n1 n2 e1
AddEdge graph1 n1 n3 e2
AddEdge graph1 n1 n4 e3
AddEdge graph1 n1 n5 e4
AddEdge graph1 n1 n6 e5

# Print the nodes in the graph and the outgoing edges from n1
ListNodes graph1
ListChildren graph1 n1