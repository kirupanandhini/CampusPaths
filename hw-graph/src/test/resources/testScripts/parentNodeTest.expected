# Tests a graph with one parent node that has multiple children

# Create a graph
created graph graph1

# Add several nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
added node n4 to graph1
added node n5 to graph1
added node n6 to graph1

# Add several edges connecting n1 to other nodes
added edge e1 from n1 to n2 in graph1
added edge e2 from n1 to n3 in graph1
added edge e3 from n1 to n4 in graph1
added edge e4 from n1 to n5 in graph1
added edge e5 from n1 to n6 in graph1

# Print the nodes in the graph and the outgoing edges from n1
graph1 contains: n1 n2 n3 n4 n5 n6
the children of n1 in graph1 are: n2(e1) n3(e2) n4(e3) n5(e4) n6(e5)